packages/data-helper/src/legacy/Picker.tsx

```diff
@@ -346,7 +346,7 @@ export class Picker extends React.Component<PropType, StateType> {
 
     render() {
         const selectedPaths = this.state.selectedPaths ? this.state.selectedPaths : this.props.selectedPaths;
-        let openPaths = this.state.openPaths ? this.state.openPaths : this.props.openPaths;
+        let openPaths = this.state.openPaths || this.props.openPaths || this.props.defaultOpenPaths;
```

Comment:
just wondering, why adding defaultOpenPaths here ?

Reply:
On debug, the exception occurs here with `TypeError: Cannot read properties of undefined (reading 'map')` : https://github.com/Jahia/app-shell/blob/master/src/javascript/apollo/client.js#L62

`openPaths` seem to be missing on init from this line in the Picker query: https://github.com/Jahia/javascript-components/blob/master/packages/data-helper/src/legacy/Picker.tsx#L126

Since we're not passing openPaths on site-settings-seo ([link](https://github.com/Jahia/site-settings-seo/blob/master/src/javascript/components/Move.jsx#L163)), I decided to use `defaultOpenPaths` as a default if `openPaths` doesn't exist.

Reply:
good point. I missed this part, but I think I've also did this fix prior to fixing the null checks which I guess was then preventing this being set. Now that we fixed the null checks, maybe we don't need this anymore after all. 

I will test and find out, thanks